.github/workflows/build-deb-package.yml

```diff
@@ -25,6 +25,28 @@ jobs:
           name: aleph-vm.debian-11.deb
           path: packaging/target/aleph-vm.debian-11.deb
 
+  build_deb_debian_12:
+    name: "Build Debian Package"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Unshallow
+        run: |
+          git fetch --prune --unshallow
+          git describe --tags
```

Comment:
Use v4, and you can checkout the repository in one step with `fetch-depth: 0`. Alternatively, if you only need tags, you can use `fetch-tags: true`.

```
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''

    # Whether to fetch tags, even if fetch-depth > 0.
    # Default: false
    fetch-tags: ''

```

Reply:
Fixed in this branch and https://github.com/aleph-im/aleph-vm/pull/407 for the rest of the repo.